config/conference.ts

```diff
@@ -25,7 +25,9 @@ const presentationSubmissionsOpenFrom = moment.parseZone('2019-04-30T08:00:00+08
 const presentationSubmissionsOpenUntil = moment.parseZone('2019-06-02T23:59:59+08:00')
 const votingOpenFrom = moment.parseZone('2019-06-08T17:00:00+08:00')
 const votingOpenUntil = moment.parseZone('2019-06-16T23:59:59+08:00')
-const agendaPublishedFrom = moment.parseZone('2019-06-23T17:00:00+08:00')
+// This is not the date for 2020 - using it to hide the agenda for now.
+// TODO: Update to actual date when known
+const agendaPublishedFrom = moment.parseZone('2020-06-23T17:00:00+08:00')
```

Comment:
We need to put last years agenda in /agenda/2019, but assume that'll come in a different PR?

Reply:
yeah, the change in how the agenda works has made that step a little more difficult. Will add a task to move it across.